nltk/parse/stanford.py

```diff
@@ -400,6 +426,76 @@ def _make_tree(self, result):
         return DependencyGraph(result, top_relation_label='ROOT')
 
 
+class CoreNLPParser(CoreNLPParser):
```

Comment:
With a new import this becomes:
```python
class CoreNLPParser(corenlp.CoreNLPParser):
```

I suggest not to use `CoreNLPParser` as a name, as it's confusing.